src/constants.ts

```diff
@@ -1,4 +1,4 @@
 import { CookieConsent } from './types';
 
 export const COOKIE_CONSENT_KEY = 'USE_COOKIE_CONSENT_STATE';
-export const EMPTY_CONSENT: CookieConsent = {};
+export const EMPTY_CONSENT: CookieConsent = { necessary: true };
```

Comment:
Since this is the "default" value, I believe this one should be initialized with the necessary cookies as `true` but let me know your thoughts!

Reply:
I think we should rename `EMPTY_CONSENT` to `DEFAULT_CONSENT`, what do you think?

Reply:
Yes, makes total sense! I just renamed it to `DEFAULT_CONSENT`!

Reply:
Nice, we can merge then!